src/commands/load.js

```diff
@@ -0,0 +1,33 @@
+let path = require('path');
+
+module.exports = () => {
+	return {
+		run: function(args, api, event) {
```

Comment:
Hmmmm, interesting. Maybe reload.js missed out on the gigantic consistency refactor. Regardless it should be consistent. 
Aside from the things mentioned, good work. I can forsee what you have done being very useful :) (sorry, I was reminded that I can be very blunt in my pull requests)